src/WebJobs.Extensions.DurableTask/EndToEndTraceHelper.cs

```diff
@@ -239,6 +239,26 @@ public void ExtensionWarningEvent(string hubName, string functionName, string in
             }
         }
 
+        public void ProcessingOutOfProcPayload(
+            string functionName,
+            string taskHub,
+            string instanceId,
+            string details)
+        {
+            EtwEventSource.Instance.ProcessingOutOfProcPayload(
+                functionName,
+                taskHub,
+                LocalAppName,
+                LocalSlotName,
+                instanceId,
+                details,
+                ExtensionVersion);
+
+            this.logger.LogInformation(
```

Comment:
This should also be updated to verbose.

Reply:
Ah yes, sorry I missed this, too much multi-tasking. Fixed!